taskcluster/ci/nimbledroid/kind.yml

```diff
@@ -12,30 +12,48 @@ kind-dependencies:
     - signing
 
 
+job-defaults:
+    description: Upload APKs to Nimbledroid for performance measurement and tracking
+    run:
+        using: run-commands
+        use-caches: false
+        secrets:
+            - name: project/mobile/fenix/nimbledroid
+              key: api_key
+              path: .nimbledroid_token
+    run-on-tasks-for: []
+    treeherder:
+        kind: test
+        platform: android-all/opt
+        tier: 2
+    worker:
+        docker-image: {in-tree: nimbledroid}
+        max-run-time: 600
+    worker-type: b-android
+
 jobs:
     nightly:
         attributes:
             nightly: true
         dependencies:
             signing: signing-nightly
-        description: Upload APKs to Nimbledroid for performance measurement and tracking
         run:
             commands:
-                - [wget, {artifact-reference: '<signing/public/build/armeabi-v7a/geckoNightly/target.apk>'}, '-O', target.apk]
+            - [wget, {artifact-reference: '<signing/public/build/armeabi-v7a/geckoNightly/target.apk>'}, '-O', target.apk]
+            - [python, automation/taskcluster/upload_apk_nimbledroid.py, target.apk, .nimbledroid_token]
+
+        treeherder:
+            symbol: nightly(nimbledroid)
+    
+    production:
```

Comment:
Oh, I just noticed, `production` will only take care of the release builds. We may want to deal with the beta ones. If so, you should just need this new section:

```yaml
jobs:
    # ...
    beta:
        attributes:
            release-type: beta
        dependencies: 
            signing: signing-beta
        run:
            commands:
                - [wget, {artifact-reference: '<signing/public/build/armeabi-v7a/geckoBeta/target.apk>'}, '-O', target.apk]
                - [python, automation/taskcluster/upload_apk_nimbledroid.py, target.apk, .nimbledroid_token]
        treeherder:
            symbol: beta(nimbledroid)